deploy-service/common/src/main/java/com/pinterest/deployservice/handler/GoalAnalyst.java

```diff
@@ -612,7 +622,15 @@ void process(String envId, EnvironBean env, PingReportBean report, AgentBean age
                 // Special case when agent state is RESET, start from beginning
                 if (updateBean.getState() == AgentState.RESET) {
```

Comment:
|| updateBean.getState() == AgentState.RESET

` LOG.debug("GoalAnalyst case 1.0.2 - host {} work on the same deploy {}, but agent state is {}`

Reply:
sounds good, thanks